Packages/vcs/Lib/vtk_ui/textbox.py

```diff
@@ -47,7 +66,11 @@ def add_character(self, character):
         row = rows[self.row]
 
         if self.column >= len(row):
-            row += character
+            if self.blank:
```

Comment:
Cool! Looking forward to it. I may leave this code in, because it means that there will still be an area for users to interact with.